controllers/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -1724,6 +1724,14 @@ func TestGenerateUnreadyNotificationContent(t *testing.T) {
 	})
 }
 
+func TestfetchDomainFromUrl(t *testing.T) {
+	t.Run("test proxy url domain extraction", func(t *testing.T) {
+		domain := fetchDomainFromUrl(logger, "https://api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com")
+		require.Equal(t, "openshiftapps.com", domain)
```

Comment:
I think removing the scheme `https://` should be good enough . As we need also the other URL parts in order to identify the environment. In this case `stone-stg-host` is what defines the RHTAP environment.

So this should be good enough as expected value:
```
api-toolchain-host-operator.apps.stone-stg-host.qc0p.p1.openshiftapps.com
```

Reply:
Yeah makes sense